src/Common/tabNavigationUtils.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Handles keyboard navigation and activation of tabs within a tab component.
+ * This function is designed to be used in components where keyboard navigation
+ * between tabs is required. It supports left and right arrow keys for tab
+ * navigation, and the Enter or Space keys for activating the currently focused tab.
+ * @param event The keyboard event triggering the tab navigation.
+ * @param currentTabIndex The index of the currently focused tab.
+ * @param tabs An array of tab objects representing the tabs in the component.
+ * @param navigateTab A function to navigate to a specific tab by index.
+ * @param activateTab A function to activate the currently focused tab basically onClick function.
+ */
+import { NormalizedEventKey } from "Common/Constants";
+import { Tab } from "Explorer/Controls/Tabs/TabComponent";
+
+export const handleTabNavigation = (
```

Comment:
This file should probably be in `src/Explorer/Controls/Tabs/` since it's specific to that component.